integration-tests/fromMnemonic-send-tez.spec.ts

```
@@ -16,67 +16,79 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
       done();
     });
     it('Should create an InMemorySigner (ed25519) with the fromMnemonic method and transfer tez to an account', async (done) => {
-      const Tezos = new TezosToolkit(rpc);
-      // with all default values
-      const signer = InMemorySigner.fromMnemonic({ mnemonic });
-      Tezos.setSignerProvider(signer);
+      try {
+        const Tezos = new TezosToolkit(rpc);
+        // with all default values
+        const signer = InMemorySigner.fromMnemonic({ mnemonic });
+        Tezos.setSignerProvider(signer);
 
-      const funderPKH = await Funder.wallet.pkh();
-      const tezosPKH = await Tezos.wallet.pkh();
+        const funderPKH = await Funder.wallet.pkh();
+        const tezosPKH = await Tezos.wallet.pkh();
 
-      const fundOp = await Funder.wallet.transfer({ to: tezosPKH, amount: 2 }).send();
-      await fundOp.confirmation();
+        const fundOp = await Funder.wallet.transfer({ to: tezosPKH, amount: 2 }).send();
+        await fundOp.confirmation();
 
-      const returnOp = await Tezos.wallet.transfer({ to: funderPKH, amount: 1 }).send();
-      await returnOp.confirmation();
+        const returnOp = await Tezos.wallet.transfer({ to: funderPKH, amount: 1 }).send();
+        await returnOp.confirmation();
 
-      const status = await returnOp.status();
-      expect(status).toEqual('applied');
+        const status = await returnOp.status();
+        expect(status).toEqual('applied');
 
-      const balance = await Tezos.tz.getBalance(tezosPKH);
-      expect(Number(balance)).toBeGreaterThan(0);
+        const balance = await Tezos.tz.getBalance(tezosPKH);
+        expect(Number(balance)).toBeGreaterThan(0);
+      } catch (err) {
+        console.log({ err, mnemonic });
```

Should the error be re-thrown after the console.log to make sure that the test fails in the pipeline and that we have visibility?

---

I agree, I think we should also throw an error alongside the console.log.

If anything, it's probably better for us to throw an error with the error message instead of outputting using console.log

---

right for some reason i thought the error would still be thrown but thrown while logging. 

switch to throw the error plus the mnemonic